lib/killbill/gen/api/pagination.rb

```diff
@@ -41,7 +41,7 @@ def initialize()
 
         def to_java()
           # conversion for iterator [type = java.util.Iterator]
-          @iterator = Killbill::Plugin::Model::EnumeratorIterator.new(@iterator)
```

Comment:
You could see my change [here](https://github.com/killbill/killbill-plugin-framework-ruby/pull/50/files#diff-632ff814489142f5b67ec628c1fc0ec3R205). The iterator is already a `Killbill::Plugin::Model::EnumeratorIterator instance, so I think there is no need to do that?

Please correct me if they are irrelevant.